pkg/api/testing/backward_compatibility_test.go

```diff
@@ -159,7 +159,7 @@ func TestCompatibility_v1_PodSecurityContext(t *testing.T) {
 	}
 
 	validator := func(obj runtime.Object) field.ErrorList {
-		return validation.ValidatePodSpec(&(obj.(*api.Pod).Spec), &(obj.(*api.Pod).ObjectMeta), field.NewPath("spec"))
+		return validation.ValidatePodSpec(&(obj.(*api.Pod).Spec), &(obj.(*api.Pod).ObjectMeta), field.NewPath("spec"), validation.PodValidationOptions{})
```

Comment:
this is very slightly concerning, because we're not necessarily using the same options we do when validating actual user objects... is there a way we can construct the options using the unified helper from here? (the package deps might not line up right)